frankDoc/src/main/java/nl/nn/adapterframework/frankdoc/front/TagletFfParameterItem.java

```diff
@@ -0,0 +1,36 @@
+/* 
+Copyright 2021 WeAreFrank! 
+
+Licensed under the Apache License, Version 2.0 (the "License"); 
+you may not use this file except in compliance with the License. 
+You may obtain a copy of the License at 
+
+    http://www.apache.org/licenses/LICENSE-2.0 
+
+Unless required by applicable law or agreed to in writing, software 
+distributed under the License is distributed on an "AS IS" BASIS, 
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+See the License for the specific language governing permissions and 
+limitations under the License. 
+*/
+
+package nl.nn.adapterframework.frankdoc.front;
```

Comment:
Ik zou het fijn vinden als een iets duidelijkere packagenaam wordt gebruikt.
Aangezien het om een doclet gaat zou ik hier taglet als package naam verwachten. Dit hoeft nu niet, maar kan jij hier rekening mee houden wanneer de package namen veranderen?

Reply:
I have added this to the work for issue https://github.com/ibissource/iaf/issues/2085.